cdap-app-fabric/src/main/java/io/cdap/cdap/internal/provision/NativeProvisioner.java

```diff
@@ -70,7 +71,7 @@ public void validateProperties(Map<String, String> properties) {
   }
 
   @Override
-  public Cluster createCluster(ProvisionerContext context) {
+  public Cluster createCluster(ProvisionerContext context, ErrorCategory errorCategory) {
```

Comment:
We should not change an SPI. Put it into the context. It would also save you a lot in passing it around. Often context is already passed

Reply:
done.